### PR TITLE
Added the basis of banners

### PR DESCRIPTION
**Pull request by Brookke**
Ready for use in the 2016-site

### CODE REVIEW

**Comment by Brookke** [comment]
Can someone please take a look at this please?

File: banner.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+func (s *Session) GetLiveBanners() (banners []Banner, err error) {
+	data, err := s.apiRequest("/banner/livebanners/", []string{})
+	if err != nil {
+		return
+	}
+
+	err = json.Unmarshal(*data, &banners)
+
+	return
+
```
Comment (CaptainHayashi) [inline banner.go]: There seem to be a few too many newlines here?  No biggie, just seems a little odd

File: banner.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import (
+	"encoding/json"
+)
+
+// Banner represents the key information about banners
+type Banner struct {
+	BannerID int    `json:"banner_id"`
+	Alt      string `json:"alt"`
+	Target   string `json:"Target"`
```
Comment (CaptainHayashi) [inline banner.go]: Odd that this one's JSON tag is uppercase, is this a myradio thing?

File: banner.go (hunk 1 above)
Comment (CaptainHayashi) [inline banner.go]: Have you run some linters (gometalinter etc) on this?

File: banner.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	URL      string `json:"url"`
+}
+
+// GetLiveBanners gets the current live banners
+// and returns a sice of banners
```
Comment (CaptainHayashi) [inline banner.go]: Typo: for `sice` read `slice` :p

File: banner.go (hunk 2 above)
Comment (Brookke) [inline banner.go]: Ah yeah I missed that

File: banner.go (hunk 1 above)
Comment (Brookke) [inline banner.go]: I am using gometalinter within visual studio code

**Comment by Brookke** [comment]
@CaptainHayashi can you please give this a look over now I have made the changes :)

**Pull request merged**